core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -272,4 +273,194 @@ public static String toJsonWithoutSpecList(TableMetadata metadata) {
     }
     return writer.toString();
   }
+
+  @Test
+  public void testJsonWithPreviousMetadataLog() throws Exception {
+    Schema schema = new Schema(
+            Types.NestedField.required(1, "x", Types.LongType.get()),
```

Comment:
Indentation should be 2 spaces for a single indent and 4 spaces for continuations.